source/slang/slang-emit-wgsl.cpp

```diff
@@ -26,6 +26,39 @@
 namespace Slang
 {
 
+// In WGSL, expression of "1.0/0.0" is not allowed, it will report compile error,
```

Comment:
I don't know how solid the foundations for infinity/etc are in our other back-ends, but at least for WGSL there seems to be little/no foundation.

https://www.w3.org/TR/WGSL/#differences-from-ieee754

I suppose we can add this support just for the test here, but what are we really testing if we have basically no guarantees from WGSL?

Reply:
"Implementations may assume that overflow, infinities, and NaNs are not present at runtime."

Reply:
I am OK with us generating out of spec code that works on majority of the platforms that most of our users care about. We can promise our user nothing, but still provide a decent solution that works most of the time, instead of enforcing restrictions that just adds friction.

Reply:
I suspected that's the policy for Slang, but good to have it confirmed.

While on the subject, what's the plan in case Slang does start specifying some of this stuff?
Are we fine with breaking apps that may be relying on behavior that Slang never promised?